dcs-packaging-tool/dcs-packaging-tool-model/src/main/java/org/dataconservancy/packaging/tool/model/dprofile/PropertyValue.java

```diff
@@ -0,0 +1,30 @@
+package org.dataconservancy.packaging.tool.model.dprofile;
+
+import java.util.List;
+
+/**
+ * The value of a property.
+ */
+public interface PropertyValue {
+    /**
+     * @return The type of the property.
+     */
+    PropertyType getPropertyType();
+
+    /**
+     * @return If type is string, return string value.
+     */
+    String asString();
```

Comment:
The idea (and this should be better documented) is that the value can be one of a few different types. One such type is string. Instead of asType methods(), it may be better to have something like getTypeValue methods such as getStringValue to better suggest that there is no type coercion.

If the type does not match the method (calling asString on a DateTime), it should throw an exception.